core/src/main/java/com/linecorp/armeria/common/ConnectionEventKey.java

```diff
@@ -0,0 +1,55 @@
+package com.linecorp.armeria.common;
+
+import java.net.InetSocketAddress;
+
+import com.linecorp.armeria.common.annotation.Nullable;
+
+import io.netty.util.AttributeKey;
+
+public class ConnectionEventKey {
```

Comment:
```suggestion
public final class ConnectionEventKey {
```
Please also add copyright and Javadocs.